crates/typos-cli/src/default_types.rs

```diff
@@ -145,7 +144,7 @@ pub const DEFAULT_TYPES: &[(&str, &[&str])] = &[
         "OFL-*[0-9]*",
     ]),
     ("lilypond", &["*.ly", "*.ily"]),
-    ("lisp", &["*.el", "*.jl", "*.lisp", "*.lsp", "*.sc", "*.scm"]),
+    ("lisp", &["*.el", "*.lisp", "*.lsp", "*.sc", "*.scm"]),
```

Comment:
Explanatory note: this file was taken from ripgrep but we operate very differently.  With ripgrep, you do a run with a specific file type and it then directly loads the extensions,  We instead find the extensions and need to map that back to names to know what dictionaries to load.  We currently only support one mapping being used and have some bugs,m like this one, where we are allowing multiple to be specified when we can't handle it.